src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-troubleshooting.mdx

```diff
@@ -52,6 +52,16 @@ You can also check for integration errors by querying [NrIntegrationError events
 
 Various parts of the New Relic UI rely on the presence of specific attributes to function properly. You can use the [NRQL console](/docs/query-your-data/explore-query-data/query-builder/nrql-console/) feature in many places to check the `WHERE` or `FACET` clauses of the query for required attributes. You can also edit those clauses and re-run the query to determine whether there is data present with those attributes missing. Examples of required attributes include `service.name` and `service.instance.id`. For a more complete list of examples, see [resources](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-concepts/#resources).
 
+## OpenTelemetry log correlation isn't working
+
+### Problem
+
+You've correlated your logs with your service using `service.id`([OpenTelemetry logs: Best practices](content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs)) so you can see logs associated with traces, but you don't see any logs in the New Relic UI. In this scenario, the log data has made it to New Relic, but isn't showing up in the distributed trace UI with corresponding spans.
```

Comment:
@rhetoric101 The link isn't working for some reason. I think it's that "content" at the start.

Reply:
@paperclypse good catch! I just fixed it!